crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -664,6 +675,41 @@ fn handle_trailing_end_of_line_condition_comment<'a>(
     CommentPlacement::Default(comment)
 }
 
+/// Handles end of line comments after the `:` of an except clause
+///
+/// ```python
+/// try:
+///    ...
+/// except: # comment
+///     pass
+/// ```
+///
+/// It attaches the comment as dangling comment to the enclosing except handler.
+fn handle_trailing_end_of_line_except_comment<'a>(
+    comment: DecoratedComment<'a>,
+    _locator: &Locator,
+) -> CommentPlacement<'a> {
+    let AnyNodeRef::ExceptHandlerExceptHandler(handler) = comment.enclosing_node() else {
+        return CommentPlacement::Default(comment);
+    };
+
+    // Must be an end of line comment
+    if comment.line_position().is_own_line() {
+        return CommentPlacement::Default(comment);
+    }
+
+    let Some(first_body_statement) = handler.body.first() else {
+        debug_assert!(false); // handlers must have a body
+        return CommentPlacement::Default(comment);
+    };
```

Comment:
I believe the logic here is incorrect. It will incorrectly attach comments after the `:` but before the first statement as dangling comments

```python
try:
	pass
except:
	# leading comment but it becomes a trailing except comment
	pass
```

Would it be possible to change `handle_in_between_bodies_end_of_line_comment` to properly support except handlers (or fix the ordering of the placement handlers)?

Reply:
i can't repro this either

Reply:
Same. This is handled correctly by the 

```
if comment.line_position().is_own_line() {
    return CommentPlacement::Default(comment);
}
```
check